integrations/sonarqube/client.py

```diff
@@ -336,20 +337,13 @@ async def get_issues_by_component(
         """
         component_issues = []
         component_key = component.get("key")
-        endpoint_path = ""
-
-        if self.is_onpremise:
-            query_params = {"project": component_key}
-            endpoint_path = Endpoints.ONPREM_ISSUES
-        else:
-            query_params = {"componentKeys": component_key}
-            endpoint_path = Endpoints.SAAS_ISSUES
+        query_params = {"componentKeys": component_key}
```

Comment:
```suggestion
        if self.is_onpremise:
            query_params = {"components": component_key}
        else:
            query_params = {"componentKeys": component_key}
```

Reply:
Only components since we are using only one API